docs/conf.py

```diff
@@ -33,6 +36,11 @@
 #
 # needs_sphinx = '1.0'
 
+# Manually mock the modules to connect to the REANA server API.
+# Note that autodoc_mock_imports does not work.
+MOCK_MODULES = ["reana_commons.api_client"]
+sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
```

Comment:
Did you also try using `:exclude-members:` in `index.rst` to exclude the problematic members?

Reply:
Yes - but unfortunately it does not work as expected, because it still tries to import `current_rs_api_client`. The alternative would be to manually declare all the functions that we want to include in the docs, but I personally prefer mocking the failing import. Do you agree?